scripts/core/exp-bindless-images.yml

```diff
@@ -133,6 +163,20 @@ members:
       name: mipFilterMode
       desc: "[in] mipmap filter mode used for filtering between mipmap levels"
 --- #--------------------------------------------------------------------------
+type: struct
+desc: "Describes an interop memory resource descriptor"
+class: $xBindlessImages
+name: $x_exp_interop_memory_desc_t
+base: $x_base_desc_t
+members: []
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Describes an interop semaphore resource descriptor"
+class: $xBindlessImages
+name: $x_exp_interop_semaphore_desc_t
+base: $x_base_desc_t
+members: []
+--- #--------------------------------------------------------------------------
```

Comment:
I though we were not going to use empty structs and instead type the parameter as `void *` to avoid this chaining?
so we have something like:

```cpp
ur_result_t urBindlessImagesImportOpaqueFDexp(ur_context_handle_t ctx, ur_device_handle_t dev, size_t size, void * pDesc, ur_exp_interop_mem_handle_t *retHandle);

ur_exp_file_descriptor_t fd{
  UR_STRUCTURE_EXP_FILE_DESCRIPTOR,
  nullptr,
  FILE
};

ur_exp_interop_mem_handle_t handle;
urBindlessImagesImportOpaqueFDexp(context, device, size, &fd, &handle);

```

Reply:
Oh, that's not how I understood the converstation. I thought we only agreed to `void*` instead of `HANDLE`.

Reply:
That would be a departure to the rest of UR's extensibilty approach, I think the PR as it stands is the most consistent approach.